cou/cli.py

```diff
@@ -86,15 +86,15 @@ def get_log_level(quiet: bool = False, verbosity: int = 0) -> str:
 
 async def analyze_and_plan(
     model_name: Optional[str], backup_database: bool
-) -> tuple[Analysis, UpgradeStep]:
+) -> tuple[Analysis, BaseStep]:
```

Comment:
If we are creating separate classes for different types of steps, doesn't make sense to have a new class to be the general plan? Something like `GeneralUpgradeStep`

Reply:
We should have `UpgradePlan` and this function should refer to return that, because it does not make sense that it will return any other, e.g. Postupgrade, ...